src/components/pager/pager-navigation.spec.js

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Immutable from 'immutable';
+import 'jest-styled-components';
+import { ThemeProvider } from 'styled-components';
+import classicTheme from '../../style/themes/classic';
+import smallTheme from '../../style/themes/small';
+import Icon from '../icon';
+import PagerNavigation from './pager-navigation.component';
+import { PagerLinkStyles } from './pager.styles';
+
+const pageSizeSelectionOptions = Immutable.fromJS([
+  { id: '10', name: 10 },
+  { id: '25', name: 25 },
+  { id: '50', name: 50 }
+]);
+
+function render(props, renderType = shallow) {
+  const theme = props.theme ? props.theme : smallTheme;
```

Comment:
I'm wondering that we can rewrite it to 
`const theme = props.theme || smallTheme`

Reply:
Looks like this still needs addressing

Reply:
@edleeks87 Looks like Dawid's comment here still needs addressing too

Reply:
Ah yeah I missed it my bad, saw the 👍 and thought it was updated